EIPS/eip-draft_Non-Fungible Token with usage rights_abbrev.md

```diff
@@ -0,0 +1,168 @@
+function getApprovedUser(uint256 tokenId) external view returns (address operator);
+```
+
+### ERC-X Receiver
+
+```solidity
+function onERCXReceived(address operator, address from, uint256 itemId, uint256 layer, bytes memory data) public returns(bytes4);
+```
+
+### ERC-X Extensions
+
+Extensions here are provided to help developers build with this standard.
+
+#### 1. ERC-X Emurable	
+
+```solidity
+function tokenOfUserByIndex(address owner, uint256 index) external view returns (uint256 tokenId);
+function totalSupply() external view returns (uint256);
+function tokenOfOwnerByIndex(address owner, uint256 index) external view returns (uint256 tokenId);
+function tokenByIndex(uint256 index) external view returns (uint256);
+```
+
+#### 2. ERC721
+
+Fully compatible with ERC721
+
+```solidity
+    event Transfer(address indexed _from, address indexed _to, uint256 indexed _tokenId);
+    event Approval(address indexed _owner, address indexed _approved, uint256 indexed _tokenId);
+    event ApprovalForAll(address indexed _owner, address indexed _operator, bool _approved);
+    function balanceOf(address _owner) external view returns (uint256);
+    function ownerOf(uint256 _tokenId) external view returns (address);
+    function safeTransferFrom(address _from, address _to, uint256 _tokenId, bytes data) external payable;
+    function safeTransferFrom(address _from, address _to, uint256 _tokenId) external payable;
+    function transferFrom(address _from, address _to, uint256 _tokenId) external payable;
+    function approve(address _approved, uint256 _tokenId) external payable;
+    function setApprovalForAll(address _operator, bool _approved) external;
+    function getApproved(uint256 _tokenId) external view returns (address);
+    function isApprovedForAll(address _owner, address _operator) external view returns (bool);
+}
+
+interface ERC165 {
+    function supportsInterface(bytes4 interfaceID) external view returns (bool);
+}
+```
+
+
+
+## Rationale
+
+#### 1 . Mortgage
+
+When mortgage NFT, you only need to mortgage the right to use into the mortgage contract, leaving the right to use.The advantage of this is that you can still use your own NFT before delivery.
+
+#### 2 . Rent 
+
+Lease does not require any collateral. First, you mortgage the NFT ownership into the contract, and the tenantry leases the right to use.
+
+This has the following benefits ：
+
+- No need to mortgage assets
+- Don't worry about the risk of default
+
+## Backward compatibility
+
+This protocol is fully backward compatible with ERC721.Refer to above Specification.
+
+
+
+## Test Cases
+
+When running the tests, you need to create a test network with Ganache-CLI:
+
+```
+ganache-cli -a 15  --gasLimit=0x1fffffffffffff -e 1000000000
+```
+
+And then run the tests using Truffle: 
+
+```
+truffle test -e development
+```
+
+Powered by Truffle and Openzeppelin test helper.
+
+## Reference Implementation
+
+[Github Reposotory](https://github.com/KKimos/ERCX).
+
+
+
```

Comment:
```suggestion
```
External links should not be used.  Either inline the reference implementation, add it to `../assets/eip-4512/` or just remove this section entirely (it is optional).

Reply:
Thanks for your reply. I have made changes but need to review the new ones. Do I need to submit them at once? Still waiting for review?